examples/pointcloud_globe.html

```diff
@@ -0,0 +1,181 @@
+        <title>Point Cloud Viewer</title>
+
+        <style type="text/css">
+            html {
+                height: 100%;
+            }
+
+            body {
+                margin: 0;
+                overflow:hidden;
+                height:100%;
+            }
+
+            #viewerDiv {
+                margin : auto auto;
+                width: 100%;
+                height: 100%;
+                padding: 0;
+                display: none;
+            }
+
+            #info {
+                color: black;
+                position: absolute;
+                top: 0;
+                left: 0;
+                margin-left: 15px;
+                padding: 0.3rem;
+                z-index: 100;
+                background-color: white;
+                border: 2px solid black;
+                border-radius: 5px;
+            }
+
+            .centered {
+                margin-left: auto;
+                margin-right: auto;
+                max-width: 25rem;
+                border: 2px solid black;
+                padding: 1rem;
+                background: rgba(128, 168, 197, 1);
+                border-radius: 5px;
+                display: none;
+            }
+
+            input {
+                margin-top: 1rem;
+                font-size: 1rem;
+            }
+
+            #url {
+                width: calc(100% - 1rem);
+            }
+
+            #submit {
+                margin-left: 25%;
+                margin-right: 25%;
+                width: 50%;
+            }
+
+            /* lopocs result display */
+            table {
+                margin-top: 3rem;
+                width: 100%;
+                border-collapse: collapse;
+            }
+            th, td {
+                padding: 0.3rem;
+                text-align: center;
+            }
+            tr:nth-child(even) {
+                background-color: #c6cfd5;
+            }
+            tr:nth-child(odd) {
+                background-color: #9fc0d5;
+            }
+            #result {
+                display: none;
+            }
+            @media (max-width: 600px) {
+                #info,
+                .dg {
+                    display: none;
+                }
+            }
+
+        </style>
+
+        <meta charset="UTF-8">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+         <script type="text/javascript">
+            var renderer;
+           // var exports = {};
+        </script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="info"></div>
+
+        <div class="centered">
+            <div id="lopocs"> </div>
+            <div id="itowns"> </div>
+            <h2>Point cloud absolute URL: </h2>
+            <h4>(file.js from PotreeConverter or Lopocs url)</h4>
+             <input id="url" type="url" value="https://raw.githubusercontent.com/iTowns/iTowns2-sample-data/master/pointclouds/eglise_saint_blaise_arles/eglise_saint_blaise_arles.js">    
```

Comment:
As @gmaillet told you, you must use these new sources
https://raw.githubusercontent.com/gmaillet/dataset/master/eglise_saint_blaise_arles.js